sqlrunner/main.py

```diff
@@ -34,6 +36,26 @@ def main(args):
     write_results(results, args["output"])
 
 
+def generate_column_headers(sql_query):
+    tsql_parser = TSQL()
+
+    # filter out any empty expressions returned by parser
+    parsed = [q for q in tsql_parser.parse(sql_query) if q]
```

Comment:
Thoughts about the `not None`? Explicit is better than implicit.

Reply:
I instinctively read `if blah` as `if blah not [something falsy]` but happy to change if you think it's unclear